app/controllers/employees_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   def show
     @employee = Employee.find(params[:id])
+    @next_meeting = Employee.next_meeting(@employee)
```

Comment:
What if we made `next_meeting` a method on `Employee`? Then you could say `@employee.next_meeting` instead of having to pass it in to Employee to get it out? That would also save you an instance variable.